src/components/PluginOutput/GetOutputComponent.tsx

```diff
@@ -43,64 +37,37 @@ const PlainText = ({ bytes }: { bytes: Uint8Array }) => {
   );
 };
 
-const ImagePng = ({ bytes }: { bytes: Uint8Array }) => {
-  const data = arrayTob64(bytes);
-
-  return (
-    <div className=" rounded h-128 border-none basis-full">
-      <img
-        className="object-contain h-128 w-full "
-        id="plugin-output-area"
-        src={`data:image/png;base64,${data}`}
-        alt=""
-      />
-    </div>
-  );
-};
-const ImageJpg = ({ bytes }: { bytes: Uint8Array }) => {
-  const data = arrayTob64(bytes);
-  return (
-    <div className="rounded h-128 border-none basis-full">
-      <img
-        className="object-contain h-128 w-full"
-        id="plugin-output-area"
-        src={`data:image/jpg;base64,${data}`}
-        alt=""
-      />
-    </div>
-  );
-};
-const ImageJpeg = ({ bytes }: { bytes: Uint8Array }) => {
-  const data = arrayTob64(bytes);
-  console.log('thishappens', data);
+// Create an Image component from a mime type
+const ImageOutput = (mimeType: string): React.FC<OutputComponentProps> => {
+  return function ImageOutputComponent({ bytes }) {
```

Comment:
using named expression so the component is not anonymous in the debugger or exceptions

Reply:
Nice! This makes sense